synapse_invite_checker/rest/base.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2020,2024 Famedly
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+import re
+
+CONTACT_MANAGEMENT_API_PREFIX = "/_synapse/client/com.famedly/tim/v1"
+INFO_API_PREFIX = "/_synapse/client/com.famedly/tim/v2/tim-information"
+
+
+def invite_checker_pattern(path_regex: str, root_prefix: str):
```

Comment:
Small nitpick, but you could provide 2 convenience methods, `contact_management_pattern` and `tim_information_pattern` and make this function just an internal helper. Then you don't need to pass the prefix every time. Again, not important, but maybe slightly neater.

Reply:
completely removed in [b9dd02b](https://github.com/famedly/synapse-invite-checker/pull/33/commits/b9dd02bde2c0bd6248d0207f6bff29c8c7c4e41b)